helpers/helpers.go

```diff
@@ -162,23 +165,38 @@ func ParseCertificatePEM(certPEM []byte) (*x509.Certificate, error) {
 	} else if cert == nil {
 		return nil, cferr.New(cferr.CertificateError, cferr.DecodeFailed)
 	} else if len(rest) > 0 {
-		return nil, cferr.Wrap(cferr.CertificateError, cferr.ParseFailed, errors.New("The PEM file should contain only one certificate."))
+		return nil, cferr.Wrap(cferr.CertificateError, cferr.ParseFailed, errors.New("The PEM file should contain only one object."))
+	} else if len(cert) > 1 {
+		return nil, cferr.Wrap(cferr.CertificateError, cferr.ParseFailed, errors.New("The PKCS7 object in the PEM file should contain only one certificate"))
 	}
-	return cert, nil
+	return cert[0], nil
 }
 
-// ParseOneCertificateFromPEM attempts to parse one certificate from the top of the certsPEM,
-// which may contain multiple certs.
-func ParseOneCertificateFromPEM(certsPEM []byte) (cert *x509.Certificate, rest []byte, err error) {
+// ParseOneObjectFromPEM attempts to parse one PEM encoded certificate object,
```

Comment:
This should be renamed `ParseOneCertificateObjectFromPEM`, if it's going to be renamed to make clear that the expectation is that we're getting a certificate out of it. It probably doesn't actually need to be renamed; it would suffice to note in the commentary that it handles both raw X.509 certificates or PKCS7 certificates.